main/CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ target_include_directories(rootcling PRIVATE
         ${CMAKE_SOURCE_DIR}/core/metacling/res
         ${CMAKE_SOURCE_DIR}/core/dictgen/res
         ${CMAKE_SOURCE_DIR}/io/rootpcm/res)
+set_property(TARGET rootcling PROPERTY ENABLE_EXPORTS 1)
```

Comment:
Is there a way to reduce this to the case that need it (i.e. "android")?

Reply:
It's not just Android. We see the same error in FreeBSD (?) and it's the right thing to do given that we call dlsym to resolve a symbol from the binary (isFromRootcling of whatever that symbol is called, used in a couple of places such as TCling()).